docs/api-reference/front-commerce-remix/generate-metas.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "generateMetas"
+sidebar_position: 3
+description: "Generate Meta tags for your Front-Commerce site."
+---
+
+The `generateMetas` helper will generate metadatas for your site.
+
+The main features of this helper are :
+
+- Merge metadata with parents Metadata (defined in the parent routes /
+  `root.tsx`)
+- Will allow easier canonical generation
+
+:::tip 
+We highly recommand you to use this helper as it will make your life
+easier when playing with nested routes. However you can still use the
+[standard Remix Meta method](https://remix.run/docs/en/main/route/meta-v2) 
+:::
+
+## Usage
+
+- The first parametes takes a [Remix V2 Meta Function](https://remix.run/docs/en/main/route/meta-v2#meta-function-parameters)
+- The second parameter is the canonical path (base url will be automatically injected)
+
+```jsx
+import generateMetas from "@front-commerce/remix/generateMetas";
+
+export const meta: V2_MetaFunction = (args: any) => {
```

Comment:
Why are `args` typed as `any`?

Reply:
Doing that you loose all the typed data from `v2_MetaFunction` 

![image](https://github.com/front-commerce/developers.front-commerce.com/assets/39598117/7067dc23-b16f-4895-8242-77f824b63040)